nixos/modules/programs/pay-respects.nix

```
@@ -169,16 +169,19 @@ in
           "url"
           "model"
         ];
-    environment = mkMerge (
-      [
-        {
-          systemPackages = [ finalPackage ];
-        }
-      ]
-      ++ map (rule: {
-        etc."xdg/pay-respects/rules/${rule.command}.toml".source = generate "${rule.command}.toml" rule;
-      }) cfg.runtimeRules
-    );
+
+    environment = {
+      etc = builtins.listToAttrs (
```

```suggestion
      etc = listToAttrs (
```
Inherit from `lib` for consistency with the rest of the module.